### PR TITLE
0.18.0 refactor integration tests

### LINKED ISSUE #3116: [Concent] Integration tests - pytest compatibility
* make the tests runnable from pytest

### LINKED ISSUE #3117: [Concent] Integration tests - log reporting
In case of failures, the tests need to be able to provide the logs either by submitting them to the standard output or - possibly preferably - by having buildbot recognize them as test artifacts so they can be downloaded

### PR DESCRIPTION
**Pull request by shadeofblue**
closes #3116 
closes #3117

### CODE REVIEW

**Review by jiivan — APPROVED** [review]
LGTM

File: scripts/node_integration_tests/playbooks/__init__.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 import sys
 from argparse import ArgumentParser
-
-from scripts.concent_integration_tests.tests.playbooks.base import (
-    NodeTestPlaybook
-)
+from ..helpers import mkdatadir
```
Comment (kmazurek) [inline scripts/node_integration_tests/playbooks/__init__.py:3]: nit: might be safer to use an absolute import (same for other occurences below)

**Review by kmazurek — APPROVED** [review]
LGTM, left one minor comment
startrekok

File: scripts/node_integration_tests/playbooks/__init__.py (hunk 1 above)
Comment (shadeofblue) [reply scripts/node_integration_tests/playbooks/__init__.py:3]: hmm... now that the location of the modules is more or less stabilized I think it's more readable and concise to just use the relative ones ...

Comment (kmazurek) [reply scripts/node_integration_tests/playbooks/__init__.py:3]: ok, fair enough

**Pull request merged**